src/open-data-fabric.md

```diff
@@ -845,7 +845,7 @@ Outputs:
 - (optional) [Data](#data) that has been produced when finalizing the old [Query](#query)
 
 #### Derive Provenance
-This operations is used to trace back a set of events in the output [Dataset](#dataset) to input events that contributed to their values or their existence (see [Provenance](#provenance)).
+These operations are used to trace back a set of events in the output [Dataset](#dataset) to input events that contributed to their values or their existence (see [Provenance](#provenance)).
```

Comment:
Should be `This operation is used` as it's talking about "Derive Provenance" operation specifically.

Reply:
Fixed